test/tests/functional/pbs_multi_sched.py

```diff
@@ -647,6 +647,268 @@ def test_multi_fairshare(self):
         self.assertEquals(n.nshares, sc3_shares)
         self.assertEquals(n.usage, sc3_usage)
 
+    @timeout(300)
+    def test_fairshare_usage(self):
+        """
+        Test the schedulers fairshare usage file and
+        check the usage file is updating correctly or not
+        """
+        self.common_setup()
```

Comment:
Don't use common_setup(). This creates all the schedulers sc1,sc2 and sc3 and its set-up. Instead use only the schedulers that are needed and use its corresponding setup_scx() functions.